src/web/app/components/question-responses/grouped-responses/grouped-responses.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="row" style="border-bottom: 1px solid #DDD;"
+<div class="row" [ngClass]="isLastResponse ? '' : 'border-bottom'"
```

Comment:
This can be simply to
```suggestion
<div class="row" [class.border-bottom]="!isLastResponse"
```
if I am not wrong

Reply:
Yeap, can confirm that this gives the same intended output